docs/design.md

```diff
@@ -64,7 +64,6 @@ Components
 ### Tools
 
 - `kubectl-myso`: CLI to manipulate MySQL cluster. It provides functionalities such as:
-  - Change master manually.
```

Comment:
I think Command-line utility for manual switchover would be helpful.
Why do you want to remove it?

Reply:
This is because I thought failover is only done by creating `SwitchoverJob`.
However, I understand doing switchover with CLI is useful, and there is no strong reason to prohibit users from executing switchover with CLI.

So, I'll revert it.

Reply:
@tapih Thanks for clarifying.

Additionally, the command-line utility can wait until the master has really changed.